stage_tester/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export GEVENT_SUPPORT=True
+export OFFLINE_TOKEN=..
```

Comment:
I think all these variables are also defined in `stage_tester/dev.conf`. Why defining them in both places?

Reply:
good point , the env variables are used by the script run.sh, the dev.conf are used by locust. In the future we want both but the environment variables should overwrite the *.conf ones (this is sort of a requirement for our on-boarding with app-sre). However this is not implemented yet and locust will read only from the file ignoring the environment variables.

to give you an idea to what we want to achieve:

https://github.com/RedHatInsights/insights-results-aggregator/blob/master/config.toml#L12 this is the "address" variable

https://github.com/RedHatInsights/insights-results-aggregator/blob/master/deploy/clowdapp.yaml#L61 this overwrites the address variable

unfortunately I wasn't able to find a library like in go that has this functionality out of the box ...

Reply:
You can try [dynaconf](https://www.dynaconf.com/), it supports toml format, and multiple environments. I think env. vars. can also be used (but haven't used so far): https://www.dynaconf.com/envvars/#lazy-values